virttest/utils_v2v.py

```diff
@@ -70,6 +70,20 @@ def get_uri(self, hostname, vpx_dc=None, esx_ip=None, username=None):
         self.vpx_dc = vpx_dc
         self.esx_ip = esx_ip
         self.username = username if username else 'root'
+        self.vpx_no_username = False
+        return uri_func()
+
+    def get_uri_without_username(self, hostname, vpx_dc=None, esx_ip=None):
+        """
+        Uri dispatcher.
+
+        :param hostname: String with host name.
```

Comment:
For other parameters e.g vpx_dc,esx_ip=, it also need add doc string

Reply:
Done. Thanks for the suggestion.